api/actions/v1alpha1/addclusterinfo_types.go

```diff
@@ -56,6 +56,8 @@ type AddClusterInfoStatus struct {
 //+kubebuilder:object:root=true
 //+kubebuilder:subresource:status
 //+kubebuilder:resource:path=addclusterinfos,scope=Namespaced,shortName=aci
+//+kubebuilder:metadata:labels=metadata.labels.odigos.io/config=true
+//+kubebuilder:metadata:labels=metadata.labels.odigos.io/system-object=1
```

Comment:
I think they are opposite, should it be?
```suggestion
//+kubebuilder:metadata:labels=metadata.labels.odigos.io/config=1
//+kubebuilder:metadata:labels=metadata.labels.odigos.io/system-object=true
```